docs/helm-broker/03-01-create-addons.md

```diff
@@ -95,29 +95,27 @@ The `plans` directory must contain at least one plan. Each plan must contain the
 
 ## docs directory
 
-In the `docs` directory, provide documentation for your addon. The documentation can include Markdown documents, AsyncAPI, OData, and OpenAPI specification files. Create the `assets` directory inside the `docs` directory to store assets, such as images. The `docs` directory must contain a `meta.yaml` which provides information on how documentation for the addon is uploaded.
-As the Helm Broker is installed as a ClusterServiceBroker, documentation for addons is provided using [ClusterDocsTopics](/components/headless-cms/#custom-resource-clusterdocstopic).
+In the `docs` directory, provide documentation for your addon. The documentation can include Markdown documents, AsyncAPI, OData, and OpenAPI specification files. Create the `assets` directory inside the `docs` directory to store assets, such as images. The `docs` directory must contain a `meta.yaml` file which provides information on how documentation for the addon is uploaded.
+As you can install the Helm Broker as a ClusterServiceBroker or as a ServiceBroker, documentation for addons is provided using either [ClusterDocsTopics](/components/headless-cms/#custom-resource-clusterdocstopic) or [DocsTopics](/components/headless-cms/#custom-resource-docs-topic) custom resources, respectively.
 
 The `meta.yaml` file contains the specification of the ClusterDocsTopic. The example structure of the `meta.yaml` file looks as follows:
```

Comment:
ClusterDocsTopic or DocsTopics -:> be consistent

OR use only DocsTopics to minimalize the amount of the text

Reply:
Good point! 🤔